### PR TITLE
Language edits to smooth the general text

### PR DESCRIPTION
**Pull request by EDRi-org**
I have updated the English version for the Privacy4Kids aka Digital Defenders booklet today. Please have a look at the pull request and tell me what you think. I made several smaller language edits to the text but more importantly updated the recommended apps section at the end to reflect the tech change since. Happy to discuss and finalise it together.

### CODE REVIEW

**Comment by petterreinholdtsen** [comment]
[European Digital Rights]
> I have updated the English version for the Privacy4Kids aka Digital
> Defenders booklet today. Please have a look at the pull request and
> tell me what you think. I made several smaller language edits to the
> text but more importantly updated the recommended apps section at the
> end to reflect the tech change since. Happy to discuss and finalise it
> together.

If possible, can you please split the change into two pull requests, one
for the textual changes and another for the changes to the list of
recommended apps?

I believe the two sets of changes deserve separate and unrelated
discussions. :)

-- 
Happy hacking
Petter Reinholdtsen

**Comment by EDRi-org** [comment]
> If possible, can you please split the change into two pull requests, one for the textual changes and another for the changes to the list of recommended apps?

Done. I hope I haven't messed up things too much. I'm still learning to use git :)

File: original/en.txt [hunk 1]
```diff
@@ -501,17 +494,17 @@ It is not the only app we can use to communicate safely, but it is one of the ea
...
-• the Adessium Foundation and Open Society Foundations
+• Individual donors on GlobalGiving.com – thank you all for your contribution!
+• The Adessium Foundation and Open Society Foundations
 
 This document is distributed under a Creative Commons 2.0 Licence (CC BY 2.0)
 http://creativecommons.org/licenses/by/2.0/
 
 » QUEST 6: I SHOULD CHOOSE AS MY PASSWORD …
-[R] …a random combination of letters and special characters.
+[R] …a random combination of letters, numbers and special characters.
```
Comment (petterreinholdtsen) [inline original/en.txt:508]: Is 'digits' better than 'numbers'?

Comment (EDRi-org) [reply original/en.txt:508]: Good question. I used "numbers" because that's how Keepass calls it.

**Comment by petterreinholdtsen** [comment]
Just for the record, I am no native English speaker, and not really capable of evaluating this change.

**Review by bfg1981 — APPROVED** [review]

**Pull request merged**